.github/workflows/auto_cherry_pick.yml

```diff
@@ -66,6 +67,19 @@ jobs:
           message: ${{ needs.find-the-parent-prt-comment.outputs.prt_comment }}
           repo-token: ${{ secrets.CHERRYPICK_PAT }}
 
+      - name: is autoMerging enabled for Auto-Cherry-Pick PR's
```

Comment:
@jyejare added this for label `AutoMergeCherryPick`

Reply:
@omkarkhatavkar Added a label `AutoMerge_Cherry_Picked` different what you suggested above to keep all cherrypick labels consistent.

Reply:
Done !